src/QMCWaveFunctions/Fermion/MultiSlaterDeterminantFast.h

```diff
@@ -55,6 +55,7 @@ class MultiSlaterDeterminantFast: public WaveFunctionComponent
 
 public:
 
+  bool Optimize;
```

Comment:
What is the difference between base class Optimizable and your Optimize?

Reply:
'Optimizable' for the Multi-slater class is set to true if "orbitals" or "CI coeffecients" are going to change. 'Optimize' is for the CI coefficients specifically. So for example if checkinvariables is called and optimize is set to false, then only the Multidiracdeterminant objects will call checkin

Reply:
Is SPOSet::Optimizable a better choice? Make it controlled in the input section of SPO.

Reply:
So I didn't do that originally because SPOSet cannot be added to the TrialWavefunction as a TrialWaveFunctionComponent. It now lives inside of MultiDiracDeterminant. Is there a way to change this so that SPOSet can be added to TrialWavefunction and I can make the change you suggested?

Reply:
Sorry If my last comment is unclear. I guess my issue is that although SPOset has it's own Optimizable variable, when evaluateDerivative is called to the trialwavefunction it only touches the trialwavefunction components. So how can I get the SPOSet to be called as well?